App/Logic/SettingsLogic.swift

```diff
@@ -100,6 +100,14 @@ extension Logic.Settings {
       try await(context.dependencies.application.goTo(url: url).run())
     }
   }
+
+  /// Shows the customer support screen
+  struct ShowCustomerSupport: AppSideEffect {
+    func sideEffect(_ context: SideEffectContext<AppState, AppDependencies>) throws {
+      let ls = CustomerSupportLS()
```

Comment:
I wouldn't pass this to the navigation, as the vc uses it for "internals" only

Reply:
I expect some info to be needed in the final version (e.g. the device model, OS version, etc).